howto/troubleshoot.md

```diff
@@ -14,6 +16,8 @@ To deploy Anbox Cloud, you need a commercial subscription for Ubuntu Advantage f
 
 ### Socket permission error for `amc`
 
+*Applies to: Anbox Cloud*
```

Comment:
Is it correct that this doesn't apply to the appliance?

Reply:
Yes, this one doesn't apply to the appliance.